class.jetpack.php

```diff
@@ -376,6 +376,13 @@ class Jetpack {
 	 */
 	public static $plugin_upgrade_lock_key = 'jetpack_upgrade_lock';
 
+	/**
+	 * Holds an instance of Automattic\Jetpack\Mc_Stats
+	 *
+	 * @var Automattic\Jetpack\Mc_Stats
+	 */
+	public $stats_instance;
```

Comment:
```suggestion
	public $mc_stats_instance;
```
Can we update this to be clearer? I am concerned about using "Stats" since it can easily be confused for the Stats module.

Reply:
good point. done